config/coordinator/coordinator-config-v2.toml

```diff
@@ -0,0 +1,322 @@
+[state-manager.request-retries]
+max-retries = 5
+backoff-delay = "PT2S"
+failures-warning-threshold = 2
+
+#[state-manager.new]
+#switch-block-number-inclusive=1000
+#version = "2.2.0"
+#endpoints = ["http://shomei:8888/"]
+#request-limit-per-endpoint = 3
+#[state-manager.new.request-retries]
+#max-retries = 5
+#backoff-delay = "PT2S"
+#failures-warning-threshold = 2
+
+[type2-state-proof-provider]
+disabled = false
+endpoints = ["http://shomei-frontend:8888/"]
+l1-query-block-tag="LATEST"
+l1-polling-interval="PT12S"
+
+[type2-state-proof-provider.request-retries]
+max-retries = 3
+backoff-delay = "PT1S"
+failures-warning-threshold = 2
+
+
+[l1-finalization-monitor]
+polling-interval = "PT1S"
+l1-query-block-tag="FINALIZED"
+
+[l1-submission] # submission
+disabled = true
+[l1-submission.dynamic-gas-price-cap]
+disabled = false
+[l1-submission.dynamic-gas-price-cap.gas-price-cap-calculation]
+adjustment-constant = 25
+blob-adjustment-constant = 25
+finalization-target-max-delay = "PT32H"
+base-fee-per-gas-percentile-window = "P7D"
+base-fee-per-gas-percentile-window-leeway = "PT10M"
+base-fee-per-gas-percentile = 10
+gas-price-caps-check-coefficient = 0.9
+[l1-submission.dynamic-gas-price-cap.fee-history-fetcher]
+fetch-interval = "PT1S"
+max-block-count = 1000
+reward-percentiles = [10, 20, 30, 40, 50, 60, 70, 80, 90, 100]
+num-of-blocks-before-latest = 2
+storage-period = "P10D"
+
+[l1-submission.fallback-gas-price]
+fee-history-block-count = 10
+fee-history-reward-percentile = 15
+
+[l1-submission.blob]
+disabled = false
+submission-delay = "PT1S"
+submission-tick-interval = "PT1S"
+max-submission-transactions-per-tick = 10
+target-blobs-per-transaction=9
+db-max-blobs-to-return = 100
+[l1-submission.blob.gas]
+gas-limit = 10000000
+max-fee-per-gas-cap = 100000000000
+max-fee-per-blob-gas-cap = 100000000000
+max-priority-fee-per-gas-cap=20000000000
+# Note: prefixed with "fallback-", used when dynamic gas price is disabled or DB is not populated yet
+[l1-submission.blob.gas.fallback]
+priority-fee-per-gas-upper-bound = 20000000000 # 20 GWEI
+priority-fee-per-gas-lower-bound = 2000000000 # 2 GWEI
+
+
+[l1-submission.blob.signer]
+# Web3j/Web3signer
+type = "Web3signer"
+
+# The account with this private key is in genesis file
+[l1-submission.blob.signer.web3j]
+private-key = "0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a"
+
+[l1-submission.blob.signer.web3signer]
+endpoint = "http://web3signer:9000"
+max-pool-size = 10
+keep-alive = true
+public-key = "9d9031e97dd78ff8c15aa86939de9b1e791066a0224e331bc962a2099a7b1f0464b8bbafe1535f2301c72c2cb3535b172da30b02686ab0393d348614f157fbdb"
+
+[l1-submission.aggregation]
+disabled = false
+endpoint = "http://l1-el-node:8545"
+submission-delay = "PT1S"
+submission-tick-interval = "PT1S"
+max-submissions-per-tick = 10
+[l1-submission.aggregation.gas]
+gas-limit = 10_000_000
+max-fee-per-gas-cap = 100_000_000_000
+max-priority-fee-per-gas-cap = 10_000_000_001
```

Comment:
These two above, if follows the old config: `gas-price-cap-multiplier-for-finalization=2.0`, should be
```
max-fee-per-gas-cap = 200_000_000_000
max-priority-fee-per-gas-cap = 40_000_000_000
```